### PR TITLE
Fixing Issue https://github.com/scorelab/Bassa/issues/347   #347

### PR DESCRIPTION
**Pull request by iammosespaulr**
Add Bassa logo to dashboard #347  Issue Fixed

### CODE REVIEW

**Comment by malithsen** [comment]
Thanks for the PR :) Please attach a screenshot of the change

File: ui/src/index.html [hunk 1]
```diff
@@ -2,7 +2,8 @@
 <html class="no-js" ng-app="bassa">
   <head>
     <meta charset="utf-8">
-    <title>Bassa</title>
+    <!--title>Bassa</title-->
```
Comment (JaDogg) [inline ui/src/index.html]: Why did you comment the title ? Don't add commented code. 👍 Keep up the good work. 😄

**Comment by agentmilindu** [comment]
@malithsen @rehrumesh Can we merge this PR? Let's not block merging a PR just because of a screenshot.

**Comment by iammosespaulr** [comment]
@agentmilindu ill make a new pr for this one

**Pull request closed without merging**